bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/skonto/SkontoScreenContent.kt

```diff
@@ -0,0 +1,1052 @@
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.viewinterop.AndroidView
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.repeatOnLifecycle
+import net.gini.android.bank.sdk.R
+import net.gini.android.bank.sdk.capture.skonto.colors.SkontoScreenColors
+import net.gini.android.bank.sdk.capture.skonto.colors.section.SkontoFooterSectionColors
+import net.gini.android.bank.sdk.capture.skonto.colors.section.SkontoInfoDialogColors
+import net.gini.android.bank.sdk.capture.skonto.colors.section.SkontoInvoicePreviewSectionColors
+import net.gini.android.bank.sdk.capture.skonto.colors.section.SkontoSectionColors
+import net.gini.android.bank.sdk.capture.skonto.colors.section.WithoutSkontoSectionColors
+import net.gini.android.bank.sdk.capture.skonto.formatter.AmountFormatter
+import net.gini.android.bank.sdk.capture.skonto.formatter.SkontoDiscountPercentageFormatter
+import net.gini.android.bank.sdk.capture.skonto.mapper.toErrorMessage
+import net.gini.android.bank.sdk.capture.skonto.model.SkontoData
+import net.gini.android.bank.sdk.capture.skonto.model.SkontoEdgeCase
+import net.gini.android.bank.sdk.capture.util.currencyFormatterWithoutSymbol
+import net.gini.android.bank.sdk.transactiondocs.ui.dialog.attachdoc.AttachDocumentToTransactionDialog
+import net.gini.android.bank.sdk.util.ui.keyboardAsState
+import net.gini.android.capture.Amount
+import net.gini.android.capture.ui.components.button.filled.GiniButton
+import net.gini.android.capture.ui.components.picker.date.GiniDatePickerDialog
+import net.gini.android.capture.ui.components.switcher.GiniSwitch
+import net.gini.android.capture.ui.components.textinput.GiniTextInput
+import net.gini.android.capture.ui.components.textinput.amount.GiniAmountTextInput
+import net.gini.android.capture.ui.components.topbar.GiniTopBar
+import net.gini.android.capture.ui.components.topbar.GiniTopBarColors
+import net.gini.android.capture.ui.theme.GiniTheme
+import net.gini.android.capture.ui.theme.modifier.tabletMaxWidth
+import net.gini.android.capture.ui.theme.typography.bold
+import net.gini.android.capture.view.InjectedViewAdapterInstance
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+@Composable
+internal fun SkontoScreenContent(
+    amountFormatter: AmountFormatter,
+    navigateBack: () -> Unit,
+    navigateToHelp: () -> Unit,
+    viewModel: SkontoFragmentViewModel,
+    modifier: Modifier = Modifier,
+    screenColorScheme: SkontoScreenColors = SkontoScreenColors.colors(),
+    isBottomNavigationBarEnabled: Boolean,
+    customBottomNavBarAdapter: InjectedViewAdapterInstance<SkontoNavigationBarBottomAdapter>?,
+    navigateToInvoiceScreen: (documentId: String, infoTextLines: List<String>) -> Unit,
```

Comment:
[Suggestion] I think this should move upward (the optional parameter should be the last)